controllers/imageupdateautomation_controller.go

```diff
@@ -269,6 +259,16 @@ func (r *ImageUpdateAutomationReconciler) Reconcile(ctx context.Context, req ctr
 			return failWithError(err)
 		}
 
+		manifestsPath := tmp
+		if auto.Spec.Update.Path != "" {
+			tracelog.Info("adjusting update path according to .spec.update.path", "base", tmp, "spec-path", auto.Spec.Update.Path)
+			if p, err := securejoin.SecureJoin(tmp, auto.Spec.Update.Path); err != nil {
+				return failWithError(err)
+			} else {
+				manifestsPath = p
+			}
```

Comment:
nit: `else` is redundant when followed by a `return`.
```suggestion
			p, err := securejoin.SecureJoin(tmp, auto.Spec.Update.Path);
			if err != nil {
				return failWithError(err)
			}
			manifestsPath = p
```

Reply:
Thanks @pjbgf :+1: